librz/bp/bp.c

```diff
@@ -306,7 +306,7 @@ RZ_API int rz_bp_list(RzBreakpoint *bp, int rad) {
 			pj_ks(pj, "data", rz_str_get(b->data));
 			pj_ks(pj, "cond", rz_str_get(b->cond));
 			pj_end(pj);
-		} else if (rad) {
+		} else if (mode) {
```

Comment:
Same here, better to use a switch instead of `if`/`else` and use specific modes.